packages/react-microlink/src/components/Card/CardContent.js

```diff
@@ -29,40 +29,65 @@ export const Content = styled.div`
   padding: 10px 15px;
   min-width: 0;
   box-sizing: border-box;
-  ${({cardSize}) => isLarge(cardSize) && largeContentStyle};
+  ${({ cardSize }) => isLarge(cardSize) && largeContentStyle};
 `
 
-const Title = styled.header`
+const Url = styled(NanoClamp)`
+  font-size: 12px;
+  margin: 0;
+`
+
+const Title = styled(NanoClamp)`
   font-size: 16px;
   font-weight: bold;
   margin: 0;
-  flex-grow: 1.2;
 `
 
-const Description = styled.div`
+const Description = styled(NanoClamp)`
```

Comment:
@Kikobeats I remember now, it needs to be wrapped because of flexbox 😅

Reply:
not really because you can attach flex properties into text components.

Looks like it's necessary in order to `nanoclamp` working fine